### PR TITLE
Update to Gradle 6.4.1

### PR DESCRIPTION
**Pull request by arturbosch**

### CODE REVIEW

File: detekt-gradle-plugin/src/test/kotlin/io/gitlab/arturbosch/detekt/MultiVersionTest.kt [hunk 1]
```diff
@@ -31,7 +31,7 @@ private fun getGradleVersionsUnderTest() =
     if (getJdkVersion() < 13) {
         listOf("5.0", "6.2.2")
```
Comment (BraisGabin) [inline detekt-gradle-plugin/src/test/kotlin/io/gitlab/arturbosch/detekt/MultiVersionTest.kt:32]: Shouldn't we upgrade this one too?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.4.1-bin.zip
```
Comment (cortinico) [inline gradle/wrapper/gradle-wrapper.properties:3]:
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-6.4.1-all.zip
```

File: detekt-gradle-plugin/src/test/kotlin/io/gitlab/arturbosch/detekt/MultiVersionTest.kt (hunk 1 above)
Comment (schalkms) [inline detekt-gradle-plugin/src/test/kotlin/io/gitlab/arturbosch/detekt/MultiVersionTest.kt:32]:
```suggestion
        listOf("5.0", "6.4.1")
```

Comment (arturbosch) [reply detekt-gradle-plugin/src/test/kotlin/io/gitlab/arturbosch/detekt/MultiVersionTest.kt:32]: It's the last Gradle version which does not support Java 13+, so we can't update this one.

Comment (arturbosch) [reply detekt-gradle-plugin/src/test/kotlin/io/gitlab/arturbosch/detekt/MultiVersionTest.kt]: Maybe I'm wrong here. Just tested it. Locally it works. I've merged @schalkms suggestion and see what the CI says.

**Pull request merged**